src/imageResponse.js

```diff
@@ -95,9 +95,11 @@ export default {
       return;
     }
 
-    // Image exists and is within bounds
+    // Image exists and is within bounds.
+    // This method is mainly used by browsers to serve retina images
     if (isHeadRequest(method)) {
       response.status(200).end();
+      log('info', `HEAD request for ${params.name}.${params.type} (${params.width}x${params.height}px, fit: ${params.fit}, blur: ${Boolean(params.blur)}) can be served`);  //eslint-disable-line max-len
```

Comment:
No I want to log this as well, to see how many of such requests we actually receive